recipes/gfortran_impl_osx-64/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set gfortran_version = [7, 3, 0] %}
+{% set libgfortran_version = [4, 0, 0] %}
+{% set max_libgfortran_version = "5.0.0.a0" %}
+{% set libquadmath_version = [0, 0, 0] %}
+{% set libgcc_s_version = [1, 0, 0] %}
+
+package:
+  name: gfortran_impl_osx-64
```

Comment:
@isuruf I just realized I put the system postfix here at the top level package name. Did you want this or only `gfortran_impl`?